recipes/depinfo/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "depinfo" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00c93bfda74090e0763c2d01a66f723f4db36dbd4e7474698b5a113c47d6a073
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pipdeptree >=0.13.2
+  run:
+    - python
+    - pip
+    - pipdeptree >=0.13.2
+
+test:
+  imports:
+    - depinfo
+
+about:
+  home: https://github.com/Midnighter/dependency-info
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: List a package's direct dependencies and their versions.
+  description: |
+    A utility Python package intended for other library packages.
+    Provides a function that when called with your package name,
+    will print platform and dependency information.
+  doc_url: https://github.com/Midnighter/dependency-info
+  dev_url: https://github.com/Midnighter/dependency-info
+
+extra:
+  recipe-maintainers:
+    - cdiener
+    - Midnighter
```

Comment:
@Midnighter Are you happy to be listed as a maintainer of this recipe?

Reply:
Pinged him on gitter and he agreed, may take a little until he replies here.

Reply:
Yes, I'm on board.